### PR TITLE
Connection solver returns delays in results.

### PR DESCRIPTION
**Pull request by tomalikem**

### CODE REVIEW

File: backend/src/main.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+if __name__ == "__main__":
+
+    downloader = Downloader()
+    vehicle_positions_pb_T, vehicle_positions_pb_A = downloader.download_vehicle_positions()
+
+    parser = VehiclePositionsParser()
+    vehicle_positions_df_T = parser.parse(vehicle_positions_pb_T)
+    vehicle_positions_df_A = parser.parse(vehicle_positions_pb_A)
+    vehicle_positions_df_A['service_id'] += 3  # service_id_offset needs stored when updating GTFS Static data
```
Comment (tomekzaw) [inline backend/src/main.py:17]: hard-coded value 3

File: backend/src/data_provider/DelayProvider.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        self.delays_producer.start()
+        print("DelaysProvider: has started.")
+        while True:
+            try:
+                downloader = Downloader()
+                vehicle_positions_pb_T, vehicle_positions_pb_A = downloader.download_vehicle_positions()
+                parser = VehiclePositionsParser()
+                vehicle_positions_df_T = parser.parse(vehicle_positions_pb_T)
+                vehicle_positions_df_A = parser.parse(vehicle_positions_pb_A)
+                vehicle_positions_df_A['service_id'] += 3  # service_id_offset needs stored when updating GTFS Static data
```
Comment (tomekzaw) [inline backend/src/data_provider/DelayProvider.py:31]: hard-coded value 3

File: backend/src/main.py (hunk 1 above)
Comment (tomekzaw) [reply backend/src/main.py:17]: BTW what does this file do?

**Review by tomekzaw — CHANGES_REQUESTED** [review]

File: backend/src/main.py (hunk 1 above)
Comment (tomalikem) [reply backend/src/main.py:17]: nothing, it shouldnt be here

**Comment by tomalikem** [comment]
closes #191

**Pull request merged**